crates/router/tests/connectors/cashtocode.rs

```diff
@@ -0,0 +1,465 @@
+use masking::Secret;
+use router::types::{self, api, storage::enums};
+
+use crate::{
+    connector_auth,
+    utils::{self, ConnectorActions},
+};
+
+#[derive(Clone, Copy)]
+struct CashtocodeTest;
+impl ConnectorActions for CashtocodeTest {}
+impl utils::Connector for CashtocodeTest {
+    fn get_data(&self) -> types::api::ConnectorData {
+        use router::connector::Cashtocode;
+        types::api::ConnectorData {
+            connector: Box::new(&Cashtocode),
+            connector_name: types::Connector::Cashtocode,
+            get_token: types::api::GetToken::Connector,
+        }
+    }
+
+    fn get_auth_token(&self) -> types::ConnectorAuthType {
+        types::ConnectorAuthType::from(
+            connector_auth::ConnectorAuthentication::new()
+                .cashtocode
+                .expect("Missing connector authentication configuration"),
+        )
+    }
+
+    fn get_name(&self) -> String {
+        "cashtocode".to_string()
+    }
+}
+
+static CONNECTOR: CashtocodeTest = CashtocodeTest {};
+
+fn get_default_payment_info() -> Option<utils::PaymentInfo> {
+    None
+}
+
+fn payment_method_details() -> Option<types::PaymentsAuthorizeData> {
+    None
+}
+
+// Cards Positive Tests
+// Creates a payment using the manual capture flow (Non 3DS).
+#[actix_web::test]
```

Comment:
I believe these tests are not applicable for cashtocode, we need to add appropiate tests for your connector

1. After you make a payment create call, you get some redirect url, check if that redirection url is present in response and status should be `AuthenticationPending`
2. Give some invalid inputs to payment create and cover the error scenarios in tests
3. Add other test scenarios also